MetaMask/metamask-extension
# Display internet protocol on the domain in SIWE screen

## Explanation

Display internet protocol (http/https) on the domain header in SIWE screen.

* Fixes #17997 

## Screenshots/Screencaps

### Before

![223092257-1275c4a0-7d58-4fe1-ab2a-1adbf6b0ac62](https://user-images.githubusercontent.com/92527393/223717507-edc92d8f-cc03-4313-8506-167759d411f0.png)

### After

![Screenshot 2023-03-08 at 13 45 30](https://user-images.githubusercontent.com/92527393/223717564-5fa663ae-1bee-4fdc-a606-d61c22448ed0.png)

## Manual Testing Steps

1. Go to https://metamask.github.io/test-dapp/
2. Connect MM
3. Click `Sign In With Ethereum` -- see how now domain **has** internet protocol http/https
